tedana/workflows/tedana.py

```diff
@@ -548,8 +548,9 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
                 LGR.warning("No BOLD components found. Re-attempting ICA.")
             elif (n_bold_comps == 0):
                 LGR.warning("No BOLD components found, but maximum number of restarts reached.")
+                keep_restarting = False
```

Comment:
Should this actually be `True`? Otherwise you'd stop here, with no BOLD components.

Reply:
Right, but at that point we are out of allotted restarts, so we have to quit.

Reply:
I'm sorry, for some reason it wasn't rendering the above `if` for me, disregard